user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -4,42 +4,52 @@ uid: I-DOCSIS_Create_CCAP_CM_pair
 
 # Creating a new CCAP/CM pair
 
-To create CCAP/CM pairs, you can either start from scratch, or start from a CSV file.
+To create CCAP/CM pairs, you can either create a single CCAP/CM pair, or create elements in bulk from a CSV file.
 
-## Creating a new CCAP/CM pair from scratch
+## Creating a single CCAP/CM pair
 
 1. In DataMiner Cube, go to apps > Automation.
 
 1. In the pane on the left, select the script *EPM_I_DOCSIS_AddNewCcapCmPair*.
 
 1. In the lower right corner, click *Execute*.
 
-   This will open a wizard with several steps.
+   This will open a pop-up window.
+
+1. Click *Create single*.
 
 1. In the **General** step:
 
-   1. Define the element name.
+   1. Define the element name (mandatory field).
 
       > [!NOTE]
       > The name of the collector element will consist of the defined element name with the suffix "_COLLECTOR".
 
-   1. Select one of the back-end DMAs for the CCAP.
+   1. Select the host for the CCAP.
 
-   1. Select one of the back-end DMAs for the collector.
+   1. Select the host for the collector.
 
```

Comment:
Starting from EPM version XXX, we support the creation of CCAP pairs (a pair consisting of the CCAP element and the CM collector element monitoring the CMs served by this CCAP) on different DMAs. However, we suggest to create the pair in the same DMA. We should proceed with creating the elements on different pairs in case of network restrictions. 

Internal note: About the EPM version, can we include a link to the RN that provides this feature?

Reply:
I've added a note with link below, but I did rephrase the note a little. Is this OK for you?

Reply:
It looks ok